src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -65,41 +73,122 @@ class Widget_List extends Widget_Abstract {
 	];
 
 	/**
-	 * @todo update in TEC-3612 & TEC-3613
-	 *
 	 * {@inheritDoc}
-	 *
-	 * @var array<string,mixed>
 	 */
-	protected $validate_arguments_map = [
-		'should_manage_url'    => 'tribe_is_truthy',
-		'no_upcoming_events'   => 'tribe_is_truthy',
-		'featured_events_only' => 'tribe_is_truthy',
-		'jsonld_enable'        => 'tribe_is_truthy',
-	];
+	protected function setup_default_arguments() {
+		$default_arguments = parent::setup_default_arguments();
+
+		$default_arguments['description'] = esc_html_x( 'A widget that displays upcoming events.', 'The description of the List Widget.', 'the-events-calendar' );
+		// @todo update name once this widget is ready to replace the existing list widget.
+		$default_arguments['name']                          = esc_html_x( 'Events List V2', 'The name of the widget.', 'the-events-calendar' );
+		$default_arguments['widget_options']['description'] = esc_html_x( 'A widget that displays upcoming events.', 'The description of the List Widget.', 'the-events-calendar' );
+		// Setup default title.
+		$default_arguments['title'] = _x( 'Upcoming Events', 'The default title of the List Widget.', 'the-events-calendar' );
+
+		return $default_arguments;
+	}
 
 	/**
```

Comment:
* :no_entry_sign: Doc comment for parameter "$new_instance" missing
* :no_entry_sign: Doc comment for parameter "$old_instance" missing

<sub>hash: bda610348cef2f355e193591b8c58b6f</sub>